### PR TITLE
Add GitHub issue and PR templates

### PR DESCRIPTION
**Pull request by RobertWSaunders**
I figured because this is going to be an open source project it would be nice for you to have a template for issues and pull request descriptions for people to follow.

Let me know if you don't and I'll make changes!

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+### List of changes:
+
+  -
+  -
+  -
+
+## Pull Request Checklist:
+
+  - [ ] My submission passes all tests.
+  - [ ] I have lint my code locally prior to submission.
```
Comment (marianoelia) [inline .github/PULL_REQUEST_TEMPLATE.md]: Lint not used in this project

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  - [ ] I have written new tests for my core changes (where applicable).
```
Comment (marianoelia) [inline .github/PULL_REQUEST_TEMPLATE.md]: No tests used.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+### List of changes:
+
+  -
+  -
+  -
+
+## Pull Request Checklist:
+
+  - [ ] My submission passes all tests.
```
Comment (marianoelia) [inline .github/PULL_REQUEST_TEMPLATE.md]: No tests used

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+1.
+
+2.
+
+3.
+
+## Specifications
+
+- Version:
+- Platform:
```
Comment (marianoelia) [inline .github/ISSUE_TEMPLATE.md]: Platform unnecessary

**Review by marianoelia — CHANGES_REQUESTED** [review]
great work. Couple changes

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (RobertWSaunders) [reply .github/PULL_REQUEST_TEMPLATE.md]: Oops, I'll change this!

File: .github/ISSUE_TEMPLATE.md (hunk 4 above)
Comment (RobertWSaunders) [reply .github/ISSUE_TEMPLATE.md]: Hmmm, maybe we'll need in this in the future though, I'll get rid of it for now!

**Comment by RobertWSaunders** [comment]
Updated!

**Review by marianoelia — APPROVED** [review]
LGTM 🚀

**Comment by RobertWSaunders** [comment]
I saw this really cool thing in the https://github.com/gatsbyjs/gatsby repo. If you try to create an issue there it first asks you if you want to file a bug, feature or have a general question. I noticed their `.github` folder has nested files, you might want to expand upon this change by adding that. I filed [this](https://github.com/gatsbyjs/gatsby/issues/8084) issue there and thought it was cool, adding to all of my projects.

**Comment by marianoelia** [comment]
Thanks I'll try it out

**Comment by marianoelia** [comment]
Checker out now

**Comment by RobertWSaunders** [comment]
Nice, you should have made a PR though! Also I noticed you committed your `.DS_STORE` files, I'd add a `.gitignore` for those suckers. I made a similar PR to add to QHacks here https://github.com/qhacks/qhacks-website/pull/95.

**Pull request merged**